packages/iris-grid/src/sidebar/ChartBuilder.test.jsx

```diff
@@ -24,19 +24,18 @@ it('renders without crashing', () => {
 });
 
 it('updates the chart type', () => {
-  const { container, unmount } = makeChartBuilderWrapper();
-
-  let chartButtons = container.querySelectorAll('.btn-chart-type');
+  const { unmount } = makeChartBuilderWrapper();
 
-  expect(chartButtons[0].classList.contains('active')).toBe(true);
-  expect(chartButtons[1].classList.contains('active')).toBe(false);
+  const lineButton = screen.getByText('Line');
+  const barButton = screen.getByText('Bar');
 
-  userEvent.click(chartButtons[1]);
+  expect(lineButton.classList.contains('active')).toBe(true);
+  expect(barButton.classList.contains('active')).toBe(false);
 
-  chartButtons = container.querySelectorAll('.btn-chart-type');
+  userEvent.click(barButton);
 
-  expect(chartButtons[0].classList.contains('active')).toBe(false);
-  expect(chartButtons[1].classList.contains('active')).toBe(true);
+  expect(lineButton.classList.contains('active')).toBe(false);
+  expect(barButton.classList.contains('active')).toBe(true);
 
   unmount();
```

Comment:
We might not need to call `unmount`. It looks like it might automatically be called by RTL since we're using Jest. https://testing-library.com/docs/react-testing-library/api#cleanup

See if it works without calling `unmount`. If you want to leave `unmount` for being explicit that's fine, but I think it's better to omit it and just let the `afterEach` handle it

Reply:
Nice good call!